build/index.js

```diff
@@ -119,6 +121,7 @@ function injectExternalLinkClasses($) {
       return;
     }
     $a.addClass("external");
+    $a.attr("rel", ` ${$a.attr("rel") || ""} noopener`.trim());
```

Comment:
```
const rel = ($a.attr("rel") || "").split(" ");
if (!rel.includes("noopener") {
  rel.push("noopener");
}
$a.attr("rel", rel.join(" "));
```

Reply:
If You use a Set You don't have to check with the if
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
If You use a Set You don't have to check with the if
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set